batch7crud-roll5/src/main/java/com/lftechnology/batch7crud/controller/EmployeeController.java

```diff
@@ -0,0 +1,199 @@
+            fetchData(request, response, pageNo - 1);
+        }
+
+    }
+
+    private void deleteProcess(HttpServletRequest request, HttpServletResponse response, int id) {
+        // TODO Auto-generated method stub
+        try {
+            employeeService.deleteById(id);
+
+            response.sendRedirect(employeeListUrl);
+
+        } catch (Exception e) {
+            // TODO Auto-generated catch block
+            e.printStackTrace();
+        }
+
+    }
+
+    private void edit(HttpServletRequest request, HttpServletResponse response, int id) {
+        // TODO Auto-generated method stub
+        try {
+            emp = employeeService.fetchById(id);
+            request.setAttribute("employee", emp);
+            RequestDispatcher dispatcher = request.getRequestDispatcher("/WEB-INF/views/editEmployee.jsp");
+            dispatcher.forward(request, response);
+
+        } catch (Exception e) {
+            // TODO Auto-generated catch block
+            e.printStackTrace();
+        }
+    }
+
+    private void editProcess(HttpServletRequest request, HttpServletResponse response, int id) {
+        // TODO Auto-generated method stub
+        try {
+            emp.setFirstName(request.getParameter("firstName"));
+            emp.setLastName(request.getParameter("lastName"));
+            emp.setDepartment(request.getParameter("department"));
+            emp.setAddress(request.getParameter("address"));
+
+            employeeService.edit(emp, id);
+
+            response.sendRedirect(employeeListUrl);
+
+        } catch (Exception e) {
+            // TODO Auto-generated catch block
+            e.printStackTrace();
+
+        }
+
+    }
+
+    private void create(HttpServletRequest request, HttpServletResponse response) {
+        // TODO Auto-generated method stub
+        try {
+            RequestDispatcher dispatcher = request.getRequestDispatcher("/WEB-INF/views/createEmployee.jsp");
+            dispatcher.forward(request, response);
+
+        } catch (Exception e) {
+            // TODO Auto-generated catch block
+            e.printStackTrace();
+        }
+
+    }
+
+    private void createProcess(HttpServletRequest request, HttpServletResponse response) {
+        // TODO Auto-generated method stub
+        try {
+            emp.setFirstName(request.getParameter("firstName"));
+            emp.setLastName(request.getParameter("lastName"));
+            emp.setDepartment(request.getParameter("department"));
+            emp.setAddress(request.getParameter("address"));
+
+            boolean status = employeeService.create(emp);
+
+            if (status == true) {
+                response.sendRedirect(employeeListUrl);
+
+            }
+
+        } catch (Exception e) {
+            // TODO: handle exception
+            e.printStackTrace();
+        }
+
+    }
+
+    private void fetchData(HttpServletRequest request, HttpServletResponse response, int pageNo) {
+        // TODO Auto-generated method stub
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1135)